File: L3/java 3/Rhombus.java
public class Rhombus extends Tetragon {


    private int side;
    private int angle;


    Rhombus(int side, int angle) {
        this.side = side;
        this.angle = angle;
    }


    public double area() {
        double angle1 = Math.toRadians(angle);
        return side*side*Math.sin(angle1);
    }

    public double perimeter() {
        return 4*side;
    }


}

File: L3/cpp 3/src/rhombus.cpp
#include <rhombus.hpp>
#include <cmath>


        Rhombus::Rhombus(int side, int angle) {
            this->side = side;
            this->angle = angle;
        }

        double Rhombus::area() {
            double angle1 = angle*M_PI/180;
            return side*side*sin(angle1);
        }

        double Rhombus::perimeter() {
            return 4*side;
        }

